applications/sofa/gui/headlessRecorder/HeadlessRecorder.cpp

```diff
@@ -346,7 +346,7 @@ int HeadlessRecorder::mainLoop()
         }
         else
         {
-            sleep(0.01);
+            sofa::helper::system::thread::CTime::sleep(10);
```

Comment:
Whole module only works on linux, so not sure this is really needed, even if technically more correct.

However, i believe the module is not compiled on the CI and is not compiled by default, so I assume this lacks the proper includes, namely `#include <sofa/helper/system/thread/CTime.h>`

Reply:
Just checked, it is compiled. I still have no idea where it get the include from. Resolved then.